src/tests/Microsoft.PowerFx.Connectors.Tests/Helpers/LoggingTestServer.cs

```diff
@@ -61,6 +62,24 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                 _log.AppendLine($" {headerName}: {value}");
             }
 
+            var httpContent = request?.Content;
+            if (httpContent != null)
+            {
+                if (httpContent.Headers != null)
+                {
+                    foreach (var h in httpContent.Headers)
+                    {
+                        _log.AppendLine($" [header] {h.Key}: {string.Join(", ", h.Value)}");                            
```

Comment:
>header [](http://example.com/codeflow?start=44&length=6)

super nit:  to a reader, this looks too much like a regular header. 
So at least, rename to "content-header" to avoid potential confusion.